### DISCUSSION TITLE
No way to customize and horrible order flow

### DISCUSSION THREAD

**Discussion by skinnerjoe** [woocommerce/woocommerce#45259] [Checkout Flow]
Paypal cannot be displayed with other payment methods and is only shown at the very top of the page.
Order summary is displayed near the top, not where you pay. A customer needs to see order totals, fees, shipping, etc. near the payment methods.
The order flow is terrible and there's no way to change anything. An ongoing problem. This never should have been released.

**Comment by senadir** [comment]
Hey! thank you for the feedback.

> A customer needs to see order totals, fees, shipping, etc. near the payment methods.
What are you basing your conclusion on? Our research didn't indicate this.

We're looking into making the [summary box sticky](https://github.com/woocommerce/woocommerce/issues/45285), but I don't see sense in having it right on top of the place order, that's now how an order flow usally goes.

> Paypal cannot be displayed with other payment methods and is only shown at the very top of the page.

Can you please provide a screenshot? PayPal works in 2 ways, an express payment method (alongside Apple Pay, Google Pay), and as a dedicated payment method alongside credit card payments.

**Comment by skinnerjoe** [comment]
I will use Amazon as an example of proper order flow. (The biggest marketplace must be doing something right) I added a screenshot to show how they always have the fees in plain view. ![Place-Your-Order-Amazon-com-Checkout](https://github.com/woocommerce/woocommerce/assets/21221832/acfc94f9-09cc-4bd0-8e1a-858fbf12aadd)
1. Shipping address
2. Payment method/Billing address
3. Review items and shipping method
4. Order summary (Shows product total, fees, discounts, shipping and tax)
5. Place order
There may be some confusion with my previous post. All I wanted to see in the final stage is the Product total, fees, discounts, shipping and tax. All should be clearly shown to the customer where they click submit. This is important because shipping and certain payment methods will add to or reduce the cost and subsequently change the taxes and order total.
The other user's recommendation of the sticky cart would be great so long as it works for mobile too.

We use Braintree as our payment provider and PayPal will only display at the top for express, not below with the other payment methods. There's not anything to show in a screenshot other than the PayPal option is not there. Here is the plugin which is an official partner with Braintree. (https://wordpress.org/plugins/woo-payment-gateway/) They are well aware of the issues with this block-based checkout and have contacted woo to resolve them.
